sys/include/net/gnrc/netdev2.h

```diff
@@ -73,6 +73,16 @@ typedef struct gnrc_netdev2 {
     gnrc_pktsnip_t * (*recv)(struct gnrc_netdev2 *dev);
 
     /**
+     * @brief Set the state of this device
+     */
+    int (*_set_netdev_state)(struct gnrc_netdev2 *dev, netopt_state_t devstate);
```

Comment:
Actually... If you just need this for state transitions… why do you need those methods anyway? Just call `dev->dev->driver->set()/get()` from those functions directly, like @kaspar030 suggested.